src/Infrastructure/Controller/Organization/User/DeleteUserController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Organization\User;
+
+use App\Application\CommandBusInterface;
+use App\Application\QueryBusInterface;
+use App\Application\User\Command\DeleteUserOrganizationCommand;
+use App\Application\User\Query\GetOrganizationUserQuery;
+use App\Domain\User\Exception\UserOrganizationNotFoundException;
+use App\Infrastructure\Security\Voter\OrganizationVoter;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+
+final class DeleteUserController
```

Comment:
Être rigoureux sur le naming jusqu'au bout ?

En parlant juste de "user" on peut penser que c'est un endpoint pour supprimer un utilisateur et toutes ses appartenances aux organisations, alors qu'on supprime son appartenance à l'organisation cible

```suggestion
final class DeleteOrganizationUserController
```